sonic_platform_base/sfp_base.py

```diff
@@ -304,3 +304,39 @@ def set_power_override(self, power_override, power_set):
             False if not
         """
         raise NotImplementedError
+
+    def read_eeprom(self, offset, num_bytes):
+        """
+        read eeprom specfic bytes for a given port_num and device id
+        begining from a random offset with size as num_bytes
+
+        Args:
+             offset :
+                     Integer, the offset from which the read transaction will start
+             num_bytes:
+                     Integer, the number of bytes to be read
+
+        Returns:
+            a bytearray , raw sequence of bytes which are read from the offset of size num_bytes
```

Comment:
Remove space before comma. Also, should we make this a `bytearray`, or `bytes` object?

Reply:
in python > 3 I think bytes and bytearray are fairly interchangeable so i have changed this to bytes.

Reply:
In Python 3 bytes and bytearray are fairly interchangeable, but seeing as we have not fully transitioned to Python 3 and need to retain support for Python 2 until all vendors have created Python 3 versions of their packages, we might want to err on the side of backward-compatibility. In that case, `bytearray` might be the way to go because it is basically the same between Python 2 and 3. Thoughts?

Reply:
i think if we keep just bytes, that would be like using str in python2.x, which might break things if we transition to python3.x as then bytes is not string in python > 3. 
so "bytearray" should be the way to go. I agree.